app/services/ocean_service.rb

```diff
@@ -0,0 +1,62 @@
+class OceanService
+  ISLAND_SCATTER_RADIUS = 200
+  CENTER = [500,500]
+
+  def initialize(number_of_islands)
+    @number_of_islands = number_of_islands
+  end
+
+  def call
+    { islands: islands }
+  end
+
+  def islands
+    if @number_of_islands.zero? || @number_of_islands.nil?
```

Comment:
try using the following syntax for statements like this:

`return [] if @number_of_islands.zero? || @number_of_islands.nil?`